.travis.yml

```diff
@@ -12,3 +12,18 @@ install:
 # command to run tests
 script:
   - make test
+
+before_deploy: sed -i "s/__version__ = \".*\"/__version__ = \"$TRAVIS_TAG\"/g" azkaban_cli/__version__.py
+
+deploy:
+  provider: pypi
+  user: "Your username"
```

Comment:
Please come back with environment variable... Can I use environment variables that are in Travis here?

Reply:
You can set secrets and other environment variables in travis site.

Output of my test pipeline looks like this:

```bash
$ echo $PYPI_PASSWORD
[secure]
```